server/automute_user_connect.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+func (p *Plugin) updateAutomutingOnUserConnect(userID string) (bool, error) {
+	if !p.isUsersPrimaryPlatformTeams(userID) {
+		return false, nil
+	}
+
+	automuteEnabled, err := p.enableAutomute(userID)
+	if err != nil {
+		p.API.LogError(
+			"Unable to enable automuting for user on connect",
+			"UserID", userID,
+			"Error", err,
+		)
+	} else {
+		var message string
+		if automuteEnabled {
+			message = "Enabled automuting for user on connect"
+		} else {
+			message = "Not enabling automute for user on connect"
+		}
+
+		p.API.LogDebug(
```

Comment:
Propose we log at `info` -- this is low volume, and helpful context for an event where debug logs might not be enabled.
```suggestion
		p.API.LogInfo(
```